lang/models.py

```diff
@@ -37,16 +37,21 @@ def get_user_permissions_per_language(cls, user):
 
 class TranslationCache(models.Model):
     text = models.TextField()
+    text_hash = models.CharField(max_length=64)
```

Comment:
We should make this unique.

Reply:
This will not be unique unless we include the source_language and dest_language also into the hash. (Otherwise normally we'll have 3 records with the same hash, with en>es, en>fr, en>ar.